scipy/_lib/tests/test__util.py

```diff
@@ -56,6 +56,13 @@ def test_check_random_state():
     rsi = check_random_state(None)
     assert_equal(type(rsi), np.random.RandomState)
     assert_raises(ValueError, check_random_state, 'a')
+    try:
+        # np.random.Generator is only available in numpy >= 1.17
+        rg = np.random.default_gen()
```

Comment:
the `default_gen` name was just changed in numpy master

Reply:
Sigh. Changed it to an explicit `rg = np.random.Generator(np.random.PCG64())`